executor/kava/guide.md

```diff
@@ -0,0 +1,221 @@
+# Set up
+## Kava 
+1. Clone kava [repo]( https://github.com/Kava-Labs/kava)
+2. Checkout branch `develop`
+3. Install the `kvd` and `kvcli` binaries with `make install`
+4. Initialize the blockchain with chain-id `testing`
+5. Create 2 accounts `deputy` and `user` that both have a bnb balance of 100000000000.
+6. In the genesis file located in `/contrib/testnet-5000/genesis-examples/genesis-bep3.json`, enter `deputy` address in the `deputy_address` field under bep3 params.
```

Comment:
Add a stop for copying the genesis file from there to ~/.kvd/config/genesis.json

Also, add a step for editing the genesis 'accounts' section to use whatever keys you generated for deputy and user.

Reply:
https://github.com/Kava-Labs/bep3-deputy/pull/1/commits/badeb0de0be59aea80df80664c5f2efce6b796fa